test/browser/home-page.test.jsx

```diff
@@ -0,0 +1,30 @@
+var should = require('should');
+var React = require('react/addons');
+var TestUtils = React.addons.TestUtils;
+
+var stubContext = require('./stub-context.jsx');
+var stubBlogFeedLoader = require('./stub-blog-feed-loader.js');
+var HomePage = require('../../pages/home.jsx');
+
+describe("HomePage", function() {
+  var homePage, blogSection;
+
+  beforeEach(function() {
+    homePage = stubContext.render(HomePage);
+    blogSection = stubContext.render(HomePage.BlogSection);
+  });
+
+  describe("blogSection", function() {
+    it("should display featured post", function() {
+      blogSection.getDOMNode().querySelector(".featured-post .entry-title").textContent.should.not.eql("");
+    });
+    it("should display 3 other latest posts", function() {
+      blogSection.getDOMNode().querySelectorAll(".recent-posts .post-title").length.should.equal(3);
+    });
```

Comment:
Ahh yeah yup that's what's going on. :( for now i recommend just adding another line at line 16:

```javascript
// Wait for the stub blog posts to load.
beforeEach(process.nextTick);
```

It's not super clean--ideally we'd maybe use [sinon](http://sinonjs.org/) to trigger the feed callback ourselves--but it'll do for now, hopefully. Let me know if it doesn't work!